lib/classes/CLI.js

```diff
@@ -44,6 +44,12 @@ class CLI {
     this.log = function () {};
   }
 
+  suppressLog() {
```

Comment:
What exactly was the problem with overriding the `process.stdout` via `process-utils/override-stdout`?

Ideally we should not patch internals to achieve that

Reply:
Changed the approach to `process-utils/override-stdout`

Reply:
Ok, so I believe we no longer need this extension (?)

Reply:
Correct :+1: